kratos/sources/accessor.cpp

```diff
@@ -0,0 +1,180 @@
+
+array_1d<double, 3> Accessor::GetValue(
+    const Variable<array_1d<double, 3>>& rVariable,
+    const Properties& rProperties,
+    const GeometryType& rGeometry,
+    const Vector& rShapeFunctionVector,
+    const ProcessInfo& rProcessInfo
+    ) const
+{
+    KRATOS_ERROR << "You are calling the GetValue of the virtual Accessor class..." << std::endl;
+    return array_1d<double, 3>();
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+array_1d<double, 6> Accessor::GetValue(
+    const Variable<array_1d<double, 6>>& rVariable,
+    const Properties& rProperties,
+    const GeometryType& rGeometry,
+    const Vector& rShapeFunctionVector,
+    const ProcessInfo& rProcessInfo
+    ) const
+{
+    KRATOS_ERROR << "You are calling the GetValue of the virtual Accessor class..." << std::endl;
+    return array_1d<double, 6>();
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+array_1d<double, 4> Accessor::GetValue(
+    const Variable<array_1d<double, 4>>& rVariable,
+    const Properties& rProperties,
+    const GeometryType& rGeometry,
+    const Vector& rShapeFunctionVector,
+    const ProcessInfo& rProcessInfo
+    ) const
+{
+    KRATOS_ERROR << "You are calling the GetValue of the virtual Accessor class..." << std::endl;
+    return array_1d<double, 4>();
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+array_1d<double, 9> Accessor::GetValue(
+    const Variable<array_1d<double, 9>>& rVariable,
+    const Properties& rProperties,
+    const GeometryType& rGeometry,
+    const Vector& rShapeFunctionVector,
+    const ProcessInfo& rProcessInfo
+    ) const
+{
+    KRATOS_ERROR << "You are calling the GetValue of the virtual Accessor class..." << std::endl;
+    return array_1d<double, 9>();
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+std::string Accessor::GetValue(
+    const Variable<std::string>& rVariable,
+    const Properties& rProperties,
+    const GeometryType& rGeometry,
+    const Vector& rShapeFunctionVector,
+    const ProcessInfo& rProcessInfo
+    ) const
+{
+    KRATOS_ERROR << "You are calling the GetValue of the virtual Accessor class..." << std::endl;
+    return "string";
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+Accessor::Pointer Accessor::Clone() const
+{
+    return Kratos::make_shared<Accessor>(*this);
```

Comment:
```suggestion
    KRATOS_ERROR << "You are calling the Clone of the virtual Accessor class..." << std::endl;
    return Kratos::make_shared<Accessor>(*this);
```

Reply:
There is no point in cloning the base class ryt?

Reply:
Agree.